### DISCUSSION TITLE
How to load a custom library as a GitHub/Google Drive URL parameter?

### DISCUSSION THREAD

**Discussion by CCA-CatherineWong** [jgraph/drawio#4343] [General]
I'm trying to follow the steps as given in this [blog post](https://www.drawio.com/blog/public-custom-libraries). 
I have my .xml custom library in a public GitHub repo: (https://github.com/USERNAME/drawio/blob/main/FILENAME.xml)
Then I encode my URL: (https%3A%2F%2Fgithub.com%2FUSERNAME%2Fdrawio%2Fblob%2Fmain%2FFILENAME.xml)
Then I end up with this URL: (https://app.diagrams.net/?splash=0&clibs=Uhttps%3A%2F%2Fgithub.com%2FUSERNAME%2Fdrawio%2Fblob%2Fmain%2FFILENAME.xml)

But that does not load my library when I click that link. Any tips?

**Comment by alderg** [comment]
Try click on the `RAW` button and use that URL instead.
